voice/methods/calls/postCallsCallId.md

```diff
@@ -77,6 +77,17 @@ except Exception as e:
     print(e)
 ```
 
+{% sample lang="js" %}
+
+```js
+var body = new BandwidthVoice.ApiModifyCallRequest({
+    "redirectUrl": "http://www.myapp.com/new",
+    "state": "active"
+});
+
+await voiceController.modifyCall(accountId, "callId", body);
```

Comment:
Does this have a return value?

Reply:
Ruby is wrapped in try/catch. Perhaps add that for JS when awaiting the result to remind folks of best practices. (they may want to try/catch at a higher scope, or they could forget)

Reply:
No return value for modifyCall. I'm fine doing a try/catch for the examples